src/Setup/DevDivVsix/RoslynCompilersExtension/Roslyn.Compilers.Extension.Resources.AllLangsNoEnu.swr

```diff
@@ -0,0 +1,7 @@
+folder "TBD"
+  file source=$(OutputPath)\Dlls\Microsoft.Build.Tasks.CodeAnalysis\netcoreapp2.0\$(Culture)\Microsoft.Build.Tasks.CodeAnalysis.resources.dll
```

Comment:
This resources file exists both under netcoreapp2.0 and net46 folder. Not sure which one should get included